examples/example.ipynb

```
@@ -0,0 +1,430 @@
+                "\n",
+                "class Author(models.Model):\n",
+                "    name = models.CharField(max_length=100)\n",
+                "    birth_date = models.DateField()\n",
+                "\n",
+                "    class Meta:\n",
+                "        app_label = \"tests\"\n",
+                "\n",
+                "class Publisher(models.Model):\n",
+                "    name = models.CharField(max_length=100)\n",
+                "    address = models.TextField(help_text=\"Publisher's official address\")\n",
+                "\n",
```

Ruff (0.11.9)

Missing docstring in public nested class

(D106)